Add new metric which indicates new connections acquire rpm

iperevoschikov opened the pull request.

Commit:
Add new metric which indicates new connections acquire rpm

Commit:
Remove using

AndrewKostousov commented on Cassandra.ThriftClient/Core/Metrics/CommandMetricsFactory.cs:
```diff
@@ -28,6 +28,13 @@ public static ISimpleCommandMetrics GetMetrics([NotNull] this ISimpleCommand com
+        {
+            return !settings.EnableMetrics
+                       ? (IPoolMetrics)NoOpMetrics.Instance
+                       : new PoolMetrics(GetMetricsContext(command : null, "Pool").Context(host).Context(keyspaceName));
```
Лучше слова ConnectionPool использовать

AndrewKostousov replied:
Наверное, сначала должен идти `Context(keyspaceName)`, а за ним уже `Context(host)`.

AndrewKostousov commented on Cassandra.ThriftClient/Core/Metrics/PoolMetrics.cs:
```diff
@@ -0,0 +1,21 @@
+            newConnections.Mark();
+        }
+
+        private readonly Meter newConnections;
```
Может, тут таймер использовать? И мерить еще и время установки новых соединений.

Commit:
Use timer instead of meter;
Reorder host and keyspace contexts;
Clearer metric context name 'ConnectionPool'

AndrewKostousov commented on Cassandra.ThriftClient/Core/Metrics/PoolMetrics.cs:
```diff
@@ -0,0 +1,24 @@
+
+namespace SkbKontur.Cassandra.ThriftClient.Core.Metrics
+{
+    internal class PoolMetrics : IPoolMetrics
```
Я бы предложил и в коде везде `PoolMetrics ` -> `ConnectionPoolMetrics ` переименовать. Интерфейс, класс, переменные...

Commit:
PoolMetrics -> ConnectionPoolMetrics

AndrewKostousov commented:
Надо еще не забыть в CHANGELOG про эту фичу написать.

Commit:
More renamings
